mesheryctl/internal/cli/root/system/system.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019 The Meshery Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package system
+
+import (
+	"errors"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	url     = "http://localhost:9081"
+	fileURL = "https://raw.githubusercontent.com/layer5io/meshery/master/docker-compose.yaml"
+)
+
+var systemDetails = `
+Manage the state and configuration of Meshery server, adapters, and client..
+
+Usage:
+  mesheryctl system [command]
+
+Available Commands:
+  cleanup     Clean up Meshery
+  help        Help about any command
```

Comment:
Can we change the description here to `help about system command`?

Reply:
Will do.